src/jdbc/cubrid/jdbc/jci/UJCIManager.java

```diff
@@ -107,10 +107,10 @@ static UUrlCache getUrlCache(UUrlHostKey key) {
 	}
 
 	public static UConnection connectDefault() throws SQLException {
```

Comment:
better to replace connectDefault with some proper connection name, for examle, connectSP, connectJSP, connectServer, and so on.